openstf/tasks/optimize_hyperparameters.py

```diff
@@ -13,22 +13,68 @@
     the CRON job can be found in the /k8s/CronJobs folder.
     Alternatively this code can be run directly by running::
 
-        $ python optimize_hyper_params.py
+        $ python optimize_hyperparameters.py
 
 """
+from datetime import datetime, timedelta
+
 from openstf.pipeline.optimize_hyperparameters import optimize_hyperparameters_pipeline
 from openstf.tasks.utils.predictionjobloop import PredictionJobLoop
 from openstf.tasks.utils.taskcontext import TaskContext
+from openstf.monitoring import teams
+
+MAX_AGE_HYPER_PARAMS_DAYS = 31
+
+
+def optimize_hyperparameters_task(pj: dict, context: TaskContext) -> None:
+
+    db = context.database
```

Comment:
A matter of taste I quess but defining db her is not strictly nescesarry. I personally would use context.database directly instead of db

Reply:
Its not necessary indeed. I've implemented your suggestion!